vividus-plugin-web-app-to-rest-api/src/main/resources/resources-validation-result.ftl

```diff
@@ -0,0 +1,88 @@
+<#ftl strip_whitespace=true>
+
+<!doctype html>
+<html lang="en">
+<head>
+    <meta charset="utf-8">
+    <title>Recources' check result table</title>
+    <link rel="stylesheet" href="../../css/external.css"/>
+    <link rel="stylesheet" href="../../styles.css"/>
+</head>
+<body>
+    <style>
+        .passed {
+            background-color: #DFF0D8;
+            color: #3C763D;
+        }
+        .failed {
+            background-color: #F2DEDE;
+            color: #A94442;
+        }
+        .broken {
+            background-color: #CCB3FF;
+            color: #661AFF;
+        }
+        .skipped {
+            background-color: #D6D8DB;
+        }
```

Comment:
we need to think about moving these classes to some common CSS that can be used in all attachments

Reply:
https://github.com/vividus-framework/vividus/issues/95